src/test/run-pass-fulldeps/proc-macro/auxiliary/issue-40001-plugin.rs

```diff
@@ -59,7 +60,7 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for MissingWhitelistedAttrPass {
             _ => cx.tcx.hir.expect_item(cx.tcx.hir.get_parent(id)),
         };
 
-        if !item.attrs.iter().any(|a| a.check_name("whitelisted_attr")) {
+        if attr::contains_name(&item.attrs, "whitelisted_attr") {
             cx.span_lint(MISSING_WHITELISTED_ATTR, span,
                          "Missing 'whitelited_attr' attribute");
```

Comment:
There is a typo: `whitelited` -> `whitelisted`

Reply:
Yeah I forgot `!`. I fixed the `whitelited` typo as well.